streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -514,7 +525,10 @@ int punctuate() {
      *                               or if the task producer got fenced (EOS)
      */
     int maybeCommitActiveTasksPerUserRequested() {
-        return active.maybeCommitPerUserRequested();
+        if (!rebalanceInProgress) {
```

Comment:
nit: Ternary operator seems easier to use.

Reply:
Good point